cli/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ _Released 03/14/2023 (PENDING)_
 - Fixed an issue where using `Cypress.require()` would throw the error `Cannot find module 'typescript'`. Fixes [#25885](https://github.com/cypress-io/cypress/issues/25885).
 - The [`before:spec`](https://docs.cypress.io/api/plugins/before-spec-api) API was updated to correctly support async event handlers in `run` mode. Fixes [#24403](https://github.com/cypress-io/cypress/issues/24403).
 - Updated the Component Testing community framework definition detection logic to take into account monorepo structures that hoist dependencies. Fixes [#25993](https://github.com/cypress-io/cypress/issues/25993)
+- Fixed an issue where the reporter auto-scroll configuration inside user preferences was being set implicitly. User's must now explicitly enable/disable auto-scroll under user preferences, which is enabled by default. Fixes [#24171](https://github.com/cypress-io/cypress/issues/24171) and [#25084](https://github.com/cypress-io/cypress/issues/25084)
```

Comment:
was this a regression? if so do you know when it was introduced?

Reply:
I'm glad to see this fix, but I'm not sure it lets us close those issues. For example in https://github.com/cypress-io/cypress/issues/25084 the setting being changed is a side-effect of another problem. I think it goes like this in the gif for that issue:

1. Cypress incorrectly detects user scrolling over the command log with no mouse activity when many commands are added close together. The is open-mode version of #16098, possibly caused by [false positives mentioned here](https://github.com/cypress-io/cypress/issues/25084#issuecomment-1369908583)).
1. Cypress incorrectly mutates the saved user preference for scrolling <-- the thing we are fixing now in this PR
1. User moves the mouse again and opens the settings panel to show that auto scrolling was turned off

But in #16098 what's being reported is the first problem, because the user has not intended to enter the "temporarily disable autoscrolling" state.

Reply:
@marktnoonan that's a good point. I don't think we can close #25084 (maybe note on the issue that the config should no longer be mutated?) for the false positive scroll issues, but would we be able to close #24171? I'm not sure #24171 is referring to the false positive scroll listener or the preference behavior.

Reply:
I think the comment in https://github.com/cypress-io/cypress/issues/24171 refers to the setting being turned off and the person being unaware. But the issue itself refers to "Users are reporting that the command log stops auto-scrolling during test execution unexpectedly."

We could close it as a dupe. Is there any open issue that is just about the preferences being set implicitly? That's what I would link to here, for clarity especially when reading the changelog later. We might need an new issue that isolates this.

Reply:
@marktnoonan I'll create a new issue and link it here and update the changelog

Reply:
@marktnoonan created https://github.com/cypress-io/cypress/issues/26113. will link in the PR and update the changelog